skfda/ml/classification/_depth_classifiers.py

```diff
@@ -343,7 +344,11 @@ def get_params(self, deep: bool = True) -> Mapping[str, object]:
         return params  # type: ignore[no-any-return]
 
     # Copied from scikit-learn's _BaseComposition with minor modifications
-    def _set_params(self, attr: str, **params: object) -> DDGClassifier[Input]:
+    def _set_params(
+        self,
+        attr: str,
+        **params: object,
```

Comment:
Style was left as it was, only separating the function definition in multiple lines to avoid exceeding the line limit.
I believe renaming the `params` parameter should be considered in another PR or ignored.